config/locales/en.yml

```diff
@@ -1517,6 +1516,7 @@ en:
     private and obscured contributions to this project.
   curator_identification: Curator identification
   current_flags: Current Flags
+  custom_boundary: Custom boundary
```

Comment:
Could you make this "Custom Boundary"? Most text here is going to be in title-case.